bokeh/_templates/autoload_js.js

```diff
@@ -26,31 +26,38 @@ calls it with the rendered model.
   if (typeof (window._bokeh_onload_callbacks) === "undefined"){
     window._bokeh_onload_callbacks = [];
   }
-  function load_lib(url, callback){
+  function load_libs(js_urls, callback) {
     window._bokeh_onload_callbacks.push(callback);
-    if (window._bokeh_is_loading){
+    if (window._bokeh_is_loading > 0) {
       console.log("Bokeh: BokehJS is being loaded, scheduling callback at", new Date());
       return null;
     }
     console.log("Bokeh: BokehJS not loaded, scheduling load and callback at", new Date());
-    window._bokeh_is_loading = true;
-    var s = document.createElement('script');
-    s.src = url;
-    s.async = true;
-    s.onreadystatechange = s.onload = function(){
-      {%- for file in css_files %}
-      Bokeh.embed.inject_css("{{ file }}");
-      {%- endfor %}
-      window._bokeh_onload_callbacks.forEach(function(callback){callback()});
-    };
-    s.onerror = function(){
-      console.warn("failed to load library " + url);
-    };
-    document.getElementsByTagName("head")[0].appendChild(s);
+    window._bokeh_is_loading = js_urls.length;
+    for (i = 0; i < js_urls.length; i++) {
+      url = js_urls[i];
+      var s = document.createElement('script');
+      s.src = url;
+      s.async = true;
+      s.onreadystatechange = s.onload = function() {
+        window._bokeh_is_loading--;
+        if (window._bokeh_is_loading==0) {
+          console.log("Bokeh: all BokehJS libraries loaded");
+          {%- for file in css_files %}
+          console.log("Bokeh: injecting CSS: {{ file }}");
+          Bokeh.embed.inject_css("{{ file }}");
+          {%- endfor %}
+          window._bokeh_onload_callbacks.forEach(function(callback){callback()});
```

Comment:
it would probably be good to do `delete window._bokeh_onload_callbacks` after this, since they could have a lot of stuff trapped in their closure which we'd keep from ever getting gc'd. Potentially we also want to avoid trouble if a callback is added inside the callback, to be safe against that we could do:
 ```
var callbacks = window._bokeh_onload_callbacks;
delete window._bokeh_onload_callbacks;
callbacks.forEach()
```